.gitignore

```diff
@@ -9,3 +9,4 @@
 /*.pem
 /keys
 /roles
+.devcontainer
```

Comment:
Would you mind removing this unrelated change?

Reply:
Will do